### DISCUSSION TITLE
LibreChat v0.7.0 does not support the gpt-4-turbo-2024-04-09 model ?

### DISCUSSION THREAD

**Discussion by lam584** [danny-avila/LibreChat#2408] [Q&A] [answered]
The LibreChat version I am currently using is v0.7.0, and I cannot find gpt-4-turbo-2024-04-09 in the model list.
![image](https://github.com/danny-avila/LibreChat/assets/107453273/2ea64c3a-d905-4d07-8c7d-36dccf652756)
![image](https://github.com/danny-avila/LibreChat/assets/107453273/00bd52cc-578d-4dd8-b973-52d65a50190f)

**Comment by berry-13** [comment, marked as answer]
You can specify the desired models by configuring the OPENAI_MODELS environment variable. For instance:

`OPENAI_MODELS=gpt-3.5-turbo-0125,gpt-3.5-turbo-0301,gpt-3.5-turbo,gpt-4,gpt-4-0613,gpt-4-vision-preview,gpt-3.5-turbo-0613,gpt-3.5-turbo-16k-0613,gpt-4-0125-preview,gpt-4-turbo-preview,gpt-4-1106-preview,gpt-3.5-turbo-1106,gpt-3.5-turbo-instruct,gpt-3.5-turbo-instruct-0914,gpt-3.5-turbo-16k,gpt-4-turbo-2024-04-09`

as you can see, I added `gpt-4-turbo-2024-04-09` as the last model

**Reply by lam584** [reply]
Thank you for your help.
